2025.2/config.json

```diff
@@ -0,0 +1,60 @@
+{
+  "defaultFiles": "./defaultFiles",
+  "parentComponents": [
+    {
+      "label": "Static Private App",
+      "type": "app",
+      "authType": "static",
+      "distribution": "private",
+      "path": "components/app/static-private"
+    },
+    {
+      "label": "OAuth Private App",
+      "type": "app",
+      "authType": "oauth",
+      "distribution": "private",
+      "path": "components/app/oauth-private"
+    },
+    {
+      "label": "OAuth Marketplace App",
+      "type": "app",
+      "authType": "oauth",
+      "distribution": "marketplace",
+      "path": "components/app/oauth-marketplace"
+    }
```

Comment:
It was easier to split these up than have the CLI tried to do some sort of auto-generation